README.md

```diff
@@ -824,6 +825,228 @@ and Equinox will supply the _initial_ value for the `project` function to render
+    - no major focus on blind-writes, even if there is low level support and/or things work if you go direct and do it out of band)
+    - provide a good story for managing the writing of the first event in a stream in an efficient manner
+  - have a story for providing a changefeed
+    - typically via a matching Propulsion library (fully ordered for SSS and ESDB, ordered at stream level for CosmosDB, similar for DynamoDB if/when that happens)
+  - have a story for caching and efficient usage of the store
+    - `Equinox.SqlStreamStore`
+      - caching is supported and recommended to minimise reads
+      - in-stream snapshots sometimes help but there are tradeoffs
+    - `Equinox.EventStore`
+      - caching is supported, but less important/relevant than it is with SSS as ESDB has good caching support and options
+      - Equinox in-stream snapshots sometimes help but there are tradeoffs)
+    - `Equinox.CosmosStore`
+      - etag-checked read caching (use without that is not recommended in general, though you absolutely will and should turn it off for some streams)
+      - multiple events are packed into each document (critical to avoid per document overhead - this is configurable)
+      - etag-checked RollingState access mode enables allow you to achieve optimal perf and RU cost via the same API without writing an event every time
+
+**The provision of the changefeed needs to be considered as a primary factor in the overall design if you're trying to build a general store - the nature of what you are seeking to provide (max latency and ordering guarantees etc) will be a major factor in designing the schema for how you manage the encoding and updating of the items in the store**
+
+#### Sagas?
+
+>In the system we would like to block particular bike for the user. But at the same time create a reservation for them to store all important business information. So we use and save data to the 2 different streams of data - typical ES problem. We could use event handers / sagas but it brings another level of complexity. 
+
+There will always be complexity in any interesting system that should not just be a CRUD layer over a relational DB.
+Taking particular solution patterns off the table from the off is definitely something you need to be careful to avoid.
+As an analogy: Having lots of classes in a system can make a mess. But collapsing it all into as few as possible can result in ISP and SRP violations and actually make for a hard to navigate and grok system, despite there being less files and less lines of code (aka complexity). Coupling things can sometimes keep things Simple, but can also sometimes simply couple things.
+In my personal experience
+1) Sagas and PMs can be scary, and there are not many good examples out there in an event sourcing context
```

Comment:
**QUESTION:** I think that we have already discussed that before, but where do you draw the line between Saga and PM?

Reply:
For me a saga is stateless and typically has semantics that just ensure related things happen.
A PM has state and often a richer processing workflow
Here the intent is to convey "the stuff you are avoiding to stay on the KISS side"

Reply:
Reworded to convey that intent better